docs/concepts.md

```diff
@@ -51,8 +58,325 @@ The newly provisioned resource is automatically bound to the resource claim.
+the Kubernetes pattern for Persistent Volume (PV)
+claims and classes to managed service provisioning with support for a strong
+separation of concern between app teams and cluster administrators. 
+
+App teams can choose between cloud-specific and portable services including
+managed databases, message queues, buckets, data pipelines, and more to define
+complete applications, build once, and deploy into multiple clouds using
+continuous delivery pipelines or GitOps flows. 
+
+Cluster administrators can define self-service policies and best-practice
+configurations to accelerate app delivery and improve security, so app teams can
+focus on delivering their app instead of cloud-specific infrastructure details.
+
+Crossplane supports provisioning managed Kubernetes clusters using `kubectl` using 
+claims and classes from a single control plane.
+
+Secure connectivity between managed services and managed Kubernetes clusters is also supported
+in Crossplane such that private networking can be established declaratively using
+`kubectl`.
+
+The combination of managed services, managed clusters, and secure connectivity are
+bundled into Crossplane Stacks for GCP, AWS, and Azure. Each aspect is designed
+to be interchangable, to enable choice of managed service providers and managed
+Kubernetes provider, including independent cloud offerings.
+
+Crossplane is designed to support the following types of managed services.
+
+### Infrastructure-as-a-Service (IaaS)
+Provisoining VMs could be supported in the future.
+
+### Containers-as-a-Service (CaaS)
+Currently supported for GKE, EKS, AKS.
+
+Kubernetes clusters are another type of resource that can be dynamically provisioned using a 
+generic resource claim by the application developer and an environment specific resource 
+class by the cluster administrator.
+
+### Database-as-a-Service (DBaaS)
+Currently supported for PostgreSQL, MySQL, Redis.
+
+Database managed services can be statically or dynamically provisioned by Crossplane in AWS, GCP, and Azure.
+An application developer simply has to specify their general need for a database such as MySQL, 
+without any specific knowledge of what environment that database will run in or even what 
+specific type of database it will be at runtime.
+
+The cluster administrator specifies a resource class that acts as a template with the 
+implementation details and policy specific to the environment that the generic MySQL resource is being deployed to.
+This enables the database to be dynamically provisioned at deployment time without the 
+application developer needing to know any of the details, which promotes portability and reusability.
+
+### Storage Services
+Currently supported including S3, Buckets, and Azure Blob storage. 
+
+### Networking Services
+Support for Networks, Subnets, and Firewall Rules
+
+### Load Balancing Services
+Native Kubernetes load balancing is supported. 
```

Comment:
This is a similar grey area to Cloud DNS Services below. IMO if we don't have a managed resource for it we don't "support" it in the Crossplane Services sense so much as "exist in an ecosystem that supports it".

Reply:
fixed